client/storage/index.js

```diff
@@ -13,6 +13,7 @@ const {
     createChannel,
     makePayment,
   } = require('./payments')
+const defaultConfig = require('../../resources/defaultConfig.json');
```

Comment:
@Tunji17 - each node has its own config that is merged with the default config and is available via `ctx.config` so there is no need to import the defaultConfig.json like this.